<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>pinyinTestClass</class>
 <widget class="QWidget" name="pinyinTestClass">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>430</width>
    <height>280</height>
   </rect>
  </property>
  <property name="minimumSize">
   <size>
    <width>430</width>
    <height>280</height>
   </size>
  </property>
  <property name="maximumSize">
   <size>
    <width>430</width>
    <height>280</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>pinyinTest</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout">
   <property name="spacing">
    <number>0</number>
   </property>
   <property name="leftMargin">
    <number>0</number>
   </property>
   <property name="topMargin">
    <number>0</number>
   </property>
   <property name="rightMargin">
    <number>0</number>
   </property>
   <property name="bottomMargin">
    <number>0</number>
   </property>
   <item>
    <widget class="QFrame" name="frame">
     <property name="styleSheet">
      <string notr="true">#frame
{
	background-color: rgb(0, 0, 0);
}

QLineEdit
{
	background: rgba(234, 234, 234, 0.15);
	border-radius: 16px;
	font-size: 12px;
	font-family: 微软雅黑;
	color: #FFFFFF;
	padding-left:12px;
}

QLabel
{
	font-size: 16px;
	font-family: 微软雅黑;
	color: #FFFFFF;
}</string>
     </property>
     <property name="frameShape">
      <enum>QFrame::StyledPanel</enum>
     </property>
     <property name="frameShadow">
      <enum>QFrame::Raised</enum>
     </property>
     <widget class="QLineEdit" name="lineEdit">
      <property name="geometry">
       <rect>
        <x>40</x>
        <y>50</y>
        <width>340</width>
        <height>32</height>
       </rect>
      </property>
      <property name="text">
       <string/>
      </property>
     </widget>
     <widget class="QLabel" name="label_1">
      <property name="geometry">
       <rect>
        <x>40</x>
        <y>120</y>
        <width>340</width>
        <height>32</height>
       </rect>
      </property>
      <property name="text">
       <string>首字母</string>
      </property>
     </widget>
     <widget class="QLabel" name="label_2">
      <property name="geometry">
       <rect>
        <x>40</x>
        <y>180</y>
        <width>340</width>
        <height>32</height>
       </rect>
      </property>
      <property name="text">
       <string>字母</string>
      </property>
     </widget>
    </widget>
   </item>
  </layout>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources>
  <include location="pinyinTest.qrc"/>
 </resources>
 <connections/>
</ui>
